src/node/node_state.h

```diff
@@ -292,15 +292,15 @@ namespace ccf
     //
     // funcs in state "initialized"
     //
-    NodeCreateInfo create(StartType start_type, CCFConfig& config)
+    NodeCreateInfo create(StartType start_type, CCFConfig& config_)
     {
       std::lock_guard<SpinLock> guard(lock);
       sm.expect(State::initialized);
 
-      create_node_cert(config);
-      open_frontend(ActorsType::nodes);
+      config = std::move(config_);
```

Comment:
Is this ok if the arg is not `&&`?

Reply:
This is well-defined, but idiosyncratic. By passing as `&`, the caller lets us modify the value, but probably doesn't expect it to be moved from (potentially leaving their instance as complete junk). We should pass this as `&&` if we want to move from it.

Reply:
Good spot! It would still use the move constructor but it is confusing for the caller. Fixed to now use `&&`.